miden-lib/asm/sat/internal/account.masm

```diff
@@ -376,13 +376,13 @@ export.validate_seed
         exec.is_faucet
         # => [is_faucet, modulus, pow]
 
-        exec.constants::get_faucet_seed_digest_modulus_multiplier
-        # => [modulus_mul, is_faucet, modulus, pow]
+        exec.constants::get_faucet_seed_digest_modulus_shift
+        # => [modulus_shift, is_faucet, modulus, pow]
 
-        mul mul swap
+        mul u32unchecked_shl swap
         # => [pow, modulus]
```

Comment:
I think the other alternative could be to use a [cdrop](https://0xpolygonmiden.github.io/miden-vm/user_docs/assembly/stack_manipulation.html#conditional-manipulation) instruction. Assuming the stack looks like:
```
[is_faucet, faucet_modulus, acct_modulus, pow, ...]
```
`cdrop` should transform this to:
```
[modulus, pow, ...]
```
Where ` modulus` is either `acct_modulus` or `faucet_modulus`.

Reply:
good idea! Let's go with this solution.